node_modules/oae-core/groupprofile/bundles/default.properties

```diff
@@ -0,0 +1,3 @@
+ADD_A_DESCRIPTION_TO_THIS_GROUP = Add a description to this group
+NO_DESCRIPTION_PROVIDED_YET = No description provided yet
```

Comment:
Just show the members
![nonmanageraboutempty](https://cloud.githubusercontent.com/assets/1489257/7047421/56e4c93c-de04-11e4-84de-4b5f917603e7.png)